services/wallet/favourite.go

```diff
@@ -44,3 +44,8 @@ func (fm *FavouriteManager) AddFavourite(favourite Favourite) error {
 	_, err = insert.Exec(favourite.Address, favourite.Name)
 	return err
 }
+
+func (fm *FavouriteManager) DeleteFavourite(address common.Address) error {
```

Comment:
Worth soft deleting? 
Otherwise it's difficult to sync them

Reply:
Yes. and maybe also consider triggering sync of wallet in response to deletion.

Reply:
@cammellos not sure what you mean by this. To mark them as hidden in DB? Won't this generate as a side effect having a hiden history of favorites?